src/Elmah.Io.ElasticSearch/Elmah.Io.ElasticSearch.nuspec

```diff
@@ -2,21 +2,23 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <version>$version$</version>
-    <authors>elmah.io,jhilden</authors>
+    <authors>elmah.io,jhilden,Asgeir</authors>
```

Comment:
think my name can be removed ;)

Reply:
You did the changes man, you don't want your name on the package ?